core/src/components/datetime/test/multiple/datetime.e2e.ts

```diff
@@ -158,10 +158,16 @@ configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, config }) => {
       });
     });
 
-    test('multiple default values across months should display at least one value', async () => {
-      const datetime = await datetimeFixture.goto(config, MULTIPLE_DATES_SEPARATE_MONTHS);
+    test('should not scroll to new month when value is updated with dates in different months', async ({ page }) => {
```

Comment:
Why are we removing the old test in favor of this new one?

Reply:
The old test checked whether the datetime scrolled to one of the selected months if multiple values from different months were provided. We're removing precisely that behavior, so the test would fail if kept.